debian/libpam-policycache.postinst

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+
+DEFAULT_CACHE_STORAGE='/var/cache/libpam-policycache'
+
+case "$1" in
+  configure)
+    chmod 0700 "$DEFAULT_CACHE_STORAGE"
+    chown root:root /usr/bin/pam-escalate-helper
+    chmod 6755 /usr/bin/pam-escalate-helper
```

Comment:
There's not much of a reason. I wanted to setgid() in the code so the group any files are created with is consistent, and using the same logic as the UID handling (setuid(geteuid())) was nifty for testing. I can change it to not call setgid(), hard code the gid passed to setgid(), or use geteuid()'s primary group ID. Do you have a preference?